### PR TITLE
Making cross button work

### PR DESCRIPTION
**Pull request by DhairyaMajmudar**
-> The cross button will now hide the  **Onboarding Tips** section after clicking 

-> Screenshot

image

cc: @seveibar

### CODE REVIEW

File: src/components/LandingHero.tsx [hunk 1]
```diff
@@ -10,56 +11,63 @@ import {
...
   LayoutGrid,
   Bot,
-} from "lucide-react"
-import { CreateNewSnippetHero } from "./CreateNewSnippetHero"
-import { Link } from "wouter"
-import { TypeBadge } from "./TypeBadge"
+} from "lucide-react";
+import { CreateNewSnippetHero } from "./CreateNewSnippetHero";
+import { Link } from "wouter";
+import { TypeBadge } from "./TypeBadge";
```
Comment (seveibar) [inline src/components/LandingHero.tsx:17]: this file needs to be formatted via biome

**Review by seveibar — APPROVED** [review]
looks good but let's address the formatting before merging

**Comment by DhairyaMajmudar** [comment]
> looks good but let's address the formatting before merging

Thanks! When I ran the command `bun run format` there were about 50 files which got formatted, let me open a PR for the same

**Comment by seveibar** [comment]
@DhairyaMajmudar woops so what happened here is the repo was formatted but your base branch was a bit behind. You could do this to fix the merge conflicts:

```
git fetch
git merge origin/main
# fix conflicts
git commit
```

you could also try to resolve conflicts inside github with "Resolve Conflicts", pull locally, then format

**Comment by seveibar** [comment]
@DhairyaMajmudar some format and typecheck issues 😭 sorry this one is taking a bit to merge because i think a lot was going on with the repo- wish i had merged on my original approval

**Comment by DhairyaMajmudar** [comment]
> @DhairyaMajmudar some format and typecheck issues 😭 sorry this one is taking a bit to merge because i think a lot was going on with the repo- wish i had merged on my original approval

Oh! 😢  I've fixed the format issues can you pls. review again

**Comment by seveibar** [comment]
The first PR to a repo is always the most annoying because of the Github workflow protection... after this is merged it should be easier!!!

**Pull request merged**